Core/Utilities/ElasticSearchDB.py

```diff
@@ -15,9 +17,51 @@
 from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.Utilities import Time
 from DIRAC.Core.Utilities import DErrno
+from DIRAC.Core.Security import Locations, X509Chain
 
 __RCSID__ = "$Id$"
 
+def getCert():
+  """
+  get the host certificate
+  """
+  cert = Locations.getHostCertificateAndKeyLocation()
+  if cert:
+    cert = cert[0]
+  else:
+    cert = "/opt/dirac/etc/grid-security/hostcert.pem"
```

Comment:
IIUC this location is checked by the `Locations.getHostCertificateAndKeyLocation()` function so it will never end up here (which makes this `getCert` useless). And, if it doesn't, then it should.